README.md

```diff
@@ -25,117 +25,120 @@ The I2Cwrapper framework consists of **four basic components.** The first two dr
+   - Modules exist as separate **include files** (e.g. [`ServoI2C_firmware.h`](firmware/ServoI2C_firmware.h) ) and are **selected** for compilation via the [`firmware_modules.h`](firmware/firmware_modules.h) file.
    - Modules don't have to worry about the I2C overhead but can **concentrate on what's important**: interpreting and reacting to the controller device's commands and requests.
    - Modules can "inject" their code at different places in the firmware, so that there is a **high degree of flexibility**.
 
 The other two basic components are for the **I2C controller's side**:
-3. The **I2Cwrapper class**, provided by the `I2Cwrapper.h` library. 
+
+1. The [**I2Cwrapper class**](https://ftjuh.github.io/I2Cwrapper/class_i2_cwrapper.html), provided by the `I2Cwrapper.h` library. 
    - Controller sketches use an object of type I2Cwrapper to **represent the target** device which handles all low level communication tasks like CRC8 checksums, error handling etc.
    - It also provides **basic functions** for target device management like changing its I2C address, setting an interrupt pin, or making it reset.
-4. **controller libraries** for each module (e.g. `ServoI2C.h`). 
-   - controller libraries use **I2Cwrapper objects** to talk to the target device.
+2. **Controller libraries** for each module (e.g. [`ServoI2C.h`](src/ServoI2C.h)). 
+   - controller libraries use **I2Cwrapper objects** (like the class `ServoI2C` in `ServerI2C.h`) to talk to the target device.
    - They implement an interface for the respective target functionality, transmit each function call to the target device, and receive the target's reply, if the command was asking for it.
 
-## Available modules
+## Available Modules
 
-Currently the following modules are available (see below for more detailed information):
+Currently the following modules are available in the [firmware subfolder](firmware) (see below for more detailed information):
 
-* **AccelStepperI2C**: Control up to eight stepper motors with acceleration control via Mike McCauley's [AccelStepper library](https://www.airspayce.com/mikem/arduino/AccelStepper/index.html) and up to two end stops per stepper via I2C. Uses a state machine and an optional controller interrupt line to prevent I2C bus clogging. Works nearly like the original library. 
-* **ServoI2C**: Control servo motors via I2C. Works just like the plain Arduino servo library.
+* **AccelStepperI2C**: Control up to eight stepper motors with acceleration control via Mike McCauley's [AccelStepper library](https://www.airspayce.com/mikem/arduino/AccelStepper/index.html), and up to two end stops per stepper. Uses a state machine and an optional controller interrupt line to prevent I2C bus clogging. Works nearly like the original library. 
+* **ServoI2C**: Control servo motors via I2C. Works just like the plain Arduino [`Servo` library](https://www.arduino.cc/reference/en/libraries/servo).
```

Comment:
Just say and I will change to:
[`Servo`](blah) library

Reply:
Yes, please do.